paasta_tools/utils.py

```diff
@@ -841,7 +842,7 @@ def _now():
 
 def remove_ansi_escape_sequences(line):
     """Removes ansi escape sequences from the given line."""
-    return no_escape.sub('', line)
+    return no_escape.sub('', line) if type(line) is str else no_escape_b.sub(b'', line)
```

Comment:
@asottile yea thats mainly why I wasn't sure if we need to keep the string support.  From having such little experience with this codebase I wasn't sure what we were expecting from the "real world".  So I would defer to whatever you guys think.